<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>F_Paiement</class>
 <widget class="QDialog" name="F_Paiement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>503</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>59</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>503</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>503</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ImagesIcones.qrc">
    <normaloff>:/Paiement.png</normaloff>:/Paiement.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="1" column="1">
    <widget class="QPushButton" name="Bt_Annuler">
     <property name="text">
      <string>&amp;Annuler</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="Bt_OK">
     <property name="text">
      <string>&amp;OK</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="styleSheet">
        <string notr="true">QGroupBox::indicator {
    width: 32px;
    height: 32px;
}
QGroupBox::indicator:checked {
    image: url(:/carte.jpg);
}</string>
       </property>
       <property name="title">
        <string>Paie&amp;ment par carte prépayée</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <widget class="QComboBox" name="CBx_Cartes"/>
        </item>
        <item row="2" column="0" colspan="3">
         <widget class="QTableWidget" name="TW_PaiementCarte">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Choisir une carte prépayée :</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QCommandLinkButton" name="Bt_SupprimerCartePaiement">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Ne plus utiliser la carte de paiement choisie&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="ImagesIcones.qrc">
            <normaloff>:/Supprimer.png</normaloff>:/Supprimer.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>15</width>
            <height>15</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="2" colspan="2">
         <widget class="QPushButton" name="Bt_AjouterCartePaiement">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Ajouter</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QGroupBox" name="GB_Panier">
       <property name="styleSheet">
        <string notr="true">QGroupBox::indicator {
    width: 32px;
    height: 32px;
}
QGroupBox::indicator:checked {
    image: url(:/Paiement.png);
}</string>
       </property>
       <property name="title">
        <string>Aut&amp;re mode de paiement</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="3">
         <widget class="QPushButton" name="Bt_AjoutPanier">
          <property name="text">
           <string>Ajouter ce reste à payer au panier</string>
          </property>
          <property name="icon">
           <iconset resource="ImagesIcones.qrc">
            <normaloff>:/Panier.png</normaloff>:/Panier.png</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="LE_ResteAPayer">
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="Lb_RestAPayer">
          <property name="text">
           <string>Reste à payer :</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="Lb_Euro">
          <property name="text">
           <string>€</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ImagesIcones.qrc"/>
 </resources>
 <connections/>
</ui>
